src/App.js

```diff
@@ -1,14 +1,18 @@
 import "./App.css";
-import { Header } from "./components/header.component/Header";
+import { Header } from "./components/Header.component/Header";
+import Home from "./pages/Home";
 import Login from "./pages/Login";
+import ProductListing from "./pages/ProductListing";
 import { SignUp } from "./pages/SignUp";
```

Comment:
As a suggestion add an index.js file and export all your components from that file.

Reply:
Okay will do that